terraform/modules/hub/s3.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_s3_bucket" "deployment_config" {
+  bucket = "gds-${var.deployment}-config"
```

Comment:
As a tip it is useful to use the `random_string` terraform module in bucket names, especially in open source repos.

It stops vexacious people bucket squatting

Reply:
I didn't know that existed. Interesting.

Reply:
```
resource "random_string" "deployment_config_bucket_name_suffix" {
  length  = 6
  special = false
  upper   = false
  number  = false
}

resource "aws_s3_bucket" "deployment_config" {
  bucket = "gds-${var.deployment}-config-${random_string.deployment_config_bucket_name_suffix.result}"
```

suggestions not smart enough for this change :(